ibis/backends/base/sql/alchemy/query_builder.py

```diff
@@ -392,6 +416,7 @@ class AlchemyCompiler(Compiler):
     difference_class = AlchemyDifference
 
     supports_indexed_grouping_keys = True
+    null_limit = sa.null()
```

Comment:
Would you mind adding a comment indicating the meaning of this compiler-level config option?

Reply:
Added.